warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -849,15 +849,31 @@ protected ASTJexlScript updateQueryTree(ScannerFactory scannerFactory, MetadataH
     protected ASTJexlScript processTree(final ASTJexlScript originalQueryTree, ShardQueryConfiguration config, Query settings, MetadataHelper metadataHelper,
                     ScannerFactory scannerFactory, QueryData queryData, QueryStopwatch timers, QueryModel queryModel) throws DatawaveQueryException {
         ASTJexlScript queryTree = originalQueryTree;
+        
```

Comment:
I missed why this should be prior to the anyfield expansion?  Is that so the value gets expanded properly in the index and then paired with the appropriate field name in #text.  Then we apply model expansion afterwards.  It would then skip the anyfield expansion?  Not saying anything is wrong, this is me thinking aloud to make sure I understand the reasoning.

Reply:
The reason this is moved prior to the anyfield expansion because the user needs to be able to specified #TEXT(value) which is equivalent to #TEXT(_ANYFIELD_, value).  The function index query expansion would then add the index query for the unfielded value which then needs to be expanded against the global index.  The model expansion has been and still is applied to the query prior to any of this.